tests/ci/run_install_shared_and_static.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+set -exo pipefail
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+print_system_and_dependency_information
+
+export CMAKE_BUILD_PARALLEL_LEVEL=${NUM_CPU_THREADS}
+
+# Set up environment.
+
+# ROOT
+#  |
+#  - AWS_LC_DIR
+#  |
+#  - SCRATCH_FOLDER
+#    |
+#    - BUILD_DIR
+#    - install-shared
+#    - install-static
+#    - install-both
+#    - MYAPP_SRC
+
+# Assumes script is executed from the root of aws-lc directory
+AWS_LC_DIR=$(pwd)
+ROOT=$(realpath ${AWS_LC_DIR}/..)
+
+SCRATCH_DIR=${ROOT}/SCRATCH_AWSLC_INSTALL_SHARED_AND_STATIC
+mkdir -p ${SCRATCH_DIR}
+rm -rf ${SCRATCH_DIR}/*
+
+function fail() {
+    echo "testf failure: $1"
+    exit 1
+}
+
+function install_aws_lc() {
+    local INSTALL_DIR=${SCRATCH_DIR}/$1
+    local BUILD_SHARED_LIBS=$2
+
+    local BUILD_DIR=${SCRATCH_DIR}/build
+    rm -rf ${BUILD_DIR}
+    ${CMAKE_COMMAND} -H${AWS_LC_DIR} -B${BUILD_DIR} -GNinja -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -DBUILD_TESTING=OFF -D${BUILD_SHARED_LIBS}
+    ${CMAKE_COMMAND} --build ${BUILD_DIR} --target install
+}
+
+# create installation with libssl.so
```

Comment:
You mainly depends on `libcrypto`, no? :)

Reply:
But `librypto` is a dependency of `libssl`, so maybe that is the motivation?

Reply:
Yes, I personally am only interested in libcrypto

But this PR tweaks both both ssl-config.cmake and crypto-config.cmake.

So I'm testing both here.

I tweaked the comments to always mention both libssl and libcrypto